functionaltest/playwright/components/methods/homemethods.ui.ts

```diff
@@ -0,0 +1,26 @@
+import { Page } from "@playwright/test"
+import { HomeLocators } from "../locators/homelocators.ui";
+
+export class HomeMethods {
+    readonly homelocators: HomeLocators;
+
+    constructor(page: Page) {
+        this.homelocators = new HomeLocators(page);
+    }
+    
+    async ClickonLogo() {
+        await this.homelocators.logo.click();
+    }
+
+    async ClickonAboutUsBtn() {
+        await this.homelocators.aboutus.click();
+    }
+
+    async ClickonWhyChamSports() {
+        await this.homelocators.whychamsports.click();
+    }
+
+    async ClickonSportsBook() {
+        await this.homelocators.sportsbookbtn.click();
+    }
+}
```

Comment:
Same here as all method names should change

Reply:
Based on your suggentions comments updated